django/test/client.py

```diff
@@ -322,10 +322,10 @@ def _get_path(self, parsed):
         if parsed[3]:
             path += str(";") + force_str(parsed[3])
         path = uri_to_iri(path).encode(UTF_8)
-        # Under Python 3, non-ASCII values in the WSGI environ are arbitrarily
+        # Non-ASCII values in the WSGI environ are arbitrarily
```

Comment:
rewrap and reword: "Replace the behavior where Non-ASCII values in the WSGI environ are arbitrarily ..."